### DISCUSSION TITLE
Forward all bounce messages to a single email address

### DISCUSSION THREAD

**Discussion by alburni** [postalserver/postal#2525] [Help with using Postal]
All is in the title, I want to forward all the bounces to a single email address.
Can I do it with address endpoint and routing ?

Sorry if this question has already been answered but I didn't found in discussions...

Thank you for your help.

**Comment by willpower232** [comment]
You would be able to create a return path route as documented here https://docs.postalserver.io/other/auto-responders-and-bounces

**Reply by alburni** [reply]
Thanks for your reply.

I had already tried several configurations, but didn't succeed :
1. New route with name `*@domain.com` and endpoint `Accept message and immediately send bounce to sender` (that was my primary objective)
2. New address endpoint (who will receive bounces because solution 1. doesnt work) and a new route with name `*@domain.com` and the address endpoint 

None of these configurations work. I see the bounces in Incoming messages, but none forwarded :(

**Reply by sinnbeck** [reply]
Did you follow this? https://docs.postalserver.io/other/auto-responders-and-bounces

>you should enter the name as \_\_returnpath\_\_
